js/views/message.js

```diff
@@ -152,7 +217,6 @@ views.NoSearchResultMessageListView = Marionette.ItemView.extend({
 	template: "#no-search-results-message-list-template",
 
 	onRender: function() {
-		$('#load-more-mail-messages').hide();
 	}
```

Comment:
It’s at https://github.com/owncloud/mail/pull/834/files#diff-df51d65ab371ac49786ad664ec2d53dfL155, in the latest state of your branch. Github only shows comments on current code.